src/AutorestSwift/Models/Complex/Schema.swift

```diff
@@ -135,25 +136,27 @@ class Schema: Codable, LanguageShortcut {
         useKey keyName: String = "schema"
     ) throws -> Schema? where Key: CodingKey {
         var schema: Schema?
-        if let keyEnum = Key(stringValue: keyName) {
+        guard let keyEnum = Key(stringValue: keyName) else {
+            fatalError("Unable to find key enum.")
+        }
+        do {
             let schemaContainer = try container.nestedContainer(keyedBy: Schema.CodingKeys.self, forKey: keyEnum)
             let type = try schemaContainer.decode(AllSchemaTypes.self, forKey: Schema.CodingKeys.type)
-
             switch type {
             case .array:
-                schema = try? container.decode(ArraySchema.self, forKey: keyEnum)
+                schema = try container.decode(ArraySchema.self, forKey: keyEnum)
             case .dictionary:
-                schema = try? container.decode(DictionarySchema.self, forKey: keyEnum)
+                schema = try container.decode(DictionarySchema.self, forKey: keyEnum)
             case .object:
-                schema = try? container.decode(ObjectSchema.self, forKey: keyEnum)
+                schema = try container.decode(ObjectSchema.self, forKey: keyEnum)
             case .number, .integer:
-                schema = try? container.decode(NumberSchema.self, forKey: keyEnum)
+                schema = try container.decode(NumberSchema.self, forKey: keyEnum)
             case .choice:
-                schema = try? container.decode(ChoiceSchema.self, forKey: keyEnum)
+                schema = try container.decode(ChoiceSchema.self, forKey: keyEnum)
             case .dateTime:
-                schema = try? container.decode(DateTimeSchema.self, forKey: keyEnum)
+                schema = try container.decode(DateTimeSchema.self, forKey: keyEnum)
             case .string:
-                schema = try? container.decode(StringSchema.self, forKey: keyEnum)
+                schema = try container.decode(StringSchema.self, forKey: keyEnum)
             case .constant:
                 schema = try? container.decode(ConstantSchema.self, forKey: keyEnum)
```

Comment:
should this be 'try' instead of 'try?' too?

Reply:
No, instead it should be the remaining cases with `try?` should be `try`, but great catch! Thank you!